planning_domains_api.py

```diff
@@ -96,13 +104,32 @@ def change_tag(tag_type, iid, tid):
     else:
         print ("Result: %s" % str(res))
 
-def simple_query(qs):
-    res = query(qs)
+def create_collection(name, description, tags, viz, ipc, formalism):
+    """Create a new collection in the database **Formatted for RDDL (2023-03-29)** """
+    attribute = ''  # Unknown for now
+    value = ''      # Same Unknown for now
+
+    params = {'user': USER_EMAIL,
+              'password': USER_TOKEN,
+              'formalism': formalism,
+              'name': name,
+              'ipc': ipc,
+              'desc': description,
+              'tags': tags,
+              'viz': viz
+    }
+    path = "/{}/collection".format(formalism)
+    res = query(path,
+          qtype='POST',
+          params = params,
+          offline=False
+          )
+
     if res['error']:
         print ("Error: %s" % res['message'])
-        return []
     else:
-        return res['result']
```

Comment:
If it's part of the API, shouldn't the result be returned as well as printed?

Reply:
That is true, I don't know why I deleted it.